### ISSUE TITLE
Internet Radio Stations List doesn't show up

### ISSUE THREAD

**Issue by lovethisshit** [nikclayton/android-squeezer#134]
Hi,

I just started to use Squeezer with my LMS/squeezeplug devices, so I didn't dig into it deeper yet. But every now and then when I try to search for Internet-Radio Stations nothing shows up (empty ListView, no categories etc.). I can't find a way to reproduce or fix it (e.g. by restart), happens randomly.

Since I would be happy to contribute to this project I will have a look into the source code, too. Meanwhile, maybe someone experienced the same?

Some Data:
Device - Motorola Moto X (2013)
Android - Version 5.1
LMS -  Version 7.7.5 (included in max2play/squeezeplug v2.12)
Setup - 1 LMS-Server (WLAN), 2 Clients (WLAN), all raspberry pi

**Comment by kaaholst** [comment]
Hi,

Thanks for the feedback.

A tip for investigating stuff like this is to use telnet:
telnet <ip adress of LMS> <CLI port of LMS>
eg: telnet 192.168.0.13 9090
You can look up the CLI-port in the LMS settings (select settings, advanced tab and "Command Line Interface" from the drop down.).

Relevant commands:
players 0 10
radios 0 10
00:04:20:28:e9:e0 local items 0 10  

00:04:20:28:e9:e0 is the id of a connected player. The id is obtained by the "players" command.

I have noticed that if you do the e.g. "local items" command, when there are no connected players, you will get an empty response, so maybe this is the explanation. If you can think of a claver way to handle and/or inform the user, then please go ahead and implement and make a pull request.

Thanks again for your interest in the project.

**Comment by fhaertig** [comment]
Hi,
sorry for not getting back here for a long time (also switched accounts in the mean time)...
I tried to analyze what you suggested and indead - it seems to happen when there's no player registered yet. This is more an issue for max2play and as a workaround, I delayed the server startup to assure that clients and network are available.
Unfortunately I don't have a nice solution about informing the user since Squeezer seems to be focused on players and a warning about e.g. "no players registered yet" would be more confusing I guess. But if this is desired, I could include it
